Dockerfile

```diff
@@ -6,19 +6,23 @@ FROM debian:jessie
 MAINTAINER Jordan Jethwa
 
 ENV DEBIAN_FRONTEND noninteractive
-ENV SERVER_URL https://localhost:4443
-ENV RUNDECK_STORAGE_PROVIDER file
-ENV RUNDECK_PROJECT_STORAGE_TYPE file
-ENV NO_LOCAL_MYSQL false
 
-RUN echo "deb http://ftp.debian.org/debian jessie-backports main" >> /etc/apt/sources.list && apt-get -qq update && apt-get -qqy upgrade && apt-get -qqy install --no-install-recommends bash openjdk-8-jre-headless supervisor procps sudo ca-certificates openssh-client mysql-server mysql-client pwgen curl git && apt-get clean
+ARG NEEDED_PACKAGES="bash openjdk-8-jre-headless supervisor procps sudo \
+ca-certificates openssh-client mysql-server mysql-client pwgen curl git"
 
-RUN curl -Lo /tmp/rundeck.deb http://dl.bintray.com/rundeck/rundeck-deb/rundeck-2.6.9-1-GA.deb
-RUN curl -Lo /tmp/rundeck-cli.deb https://github.com/rundeck/rundeck-cli/releases/download/v0.1.19/rundeck-cli_0.1.19-1_all.deb
+RUN echo "deb http://httpredir.debian.org/debian jessie-backports main" >> /etc/apt/sources.list
```

Comment:
This repo URL is better because it uses geolocation to select best mirror to be used.